postgis.rb

```diff
@@ -107,13 +112,13 @@ def install
   end
 
   def caveats
-    <<-EOS
+    <<~EOS
```

Comment:
what's the difference between `<<-` and `<<~`?

Reply:
This came over from https://github.com/Homebrew/homebrew-core/blob/master/Formula/postgis.rb, looks like `<<~` dedents the text: https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

Reply:
Looks like `<<~` strips the leading whitespace:
https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc